app/main.ts

```diff
@@ -4,6 +4,29 @@ import * as path from 'path';
 import { readExcel } from './excel-parser/read-report';
 import { saveParts } from './excel-parser/save-report';
 
+export interface PartToSave {
```

Comment:
That's not quite what I had in mind, but let it be for now. Do not declare it in main though. Create a separate folder for models